Ross-Ulbricht-Miami-Bitcoin-Conference/en/ch001-Ross-Ulbricht-Miami-Bitcoin-Conference.md

```diff
@@ -0,0 +1,465 @@
+ Sorry. This is prison. 
+ I was trying to help us move 
+ toward a freer and more equitable world. 
+ But we all know the road to 
+ hell is paid with good intentions, right? 
+ And now, here i am. 
+ I'm in hell. 
+ I want you to understand what 
+ it means to lose your freedom. 
+  
+ 7. 
+ Let me start by telling you about the hole. 
+ Goes by many names. 
+ The shoes, segregation, 
+ the box, but for me it's the hole. 
+ The hole is the prison within the prison. 
+ I once spent four months straight in the hole. 
+ Not easy for me to talk about, but i will. 
+ the hole can make you or bridge you. 
+ And there was a time when it broke me. 
+ It started with my mind racing out of control 
+ I felt like the walls were crushing in on me. 
+ Like i just had to get out of that cell. 
+ This lasted days. 
+ Then i started beating the walls 
+ and kicking the heavy metal door. 
+ Something deep inside me cried out for freedom. 
+ I couldn't accept where i was 
+ or what had happened to me. 
+  
+ 8. 
+ But eventually i realized i had to get a grip. 
+ the stress was destroying me. 
+ It may sound strange, but what saved me was gratitude. 
+ What could i be grateful for in that little cell? 
+ Well, i had to start small. 
+ I had air, right? 
+ Maybe it was stale and foul, but i had air. 
+ I had water. 
+ That didn't make me sick. 
+ Food came through the slot in the door every day. 
+ I knew i wasn't forgotten. 
+ My family i knew some day would be 
+ over and my family would still be there. 
+ I forgave all the people involved 
+ in putting me in prison. 
+ I had to. 
+  
+ 9. 
+ The anger i felt wasn't hurting them, 
+ but it was hurting me. 
+ So for the safe of my sanity, i had to let it go. 
+ I had a dream. 
+ i was in the hole. 
+ And in the dream, i was free. 
+ I was in the park. 
+ And i felt this huge relief. 
+ I wasn't in prison anymore. 
+ And i got worried. 
+ Am i out on bail or something? 
+ Or are they going to put me back in? 
+ Are they after me right now? 
+ And i started trying to get away. 
+ and the anxiety, it just woke me up. 
+ And there i was again in the hole. 
+ And it was like everything that had happened 
+ to me over all these years, it all came 
+ slamming down on me at once. 
+  
+ 10. 
+ Like life without parole, maximum security. 
+ I've been in the hole for months. 
+ There's no end in sight. 
+ I want you to understand what 
+ it means to lose your freedom. 
+ My mother this was after i was sentenced. 
+ My mother was invited to give us 
+ speaking to her and Europe. 
+ She was raising awareness about  
+ what had happened to me 
+ and was looking for help at Talk in Poland. 
+ She started to feel a bit sick 
+ and had to fly home early. 
+ So the next morning, i called 
+ my sister from the prison. 
+ And the first thing she said was, 
+ has anyone told you about mom yet? 
+ I said, what about her? 
+ And she said to me in this voice, she said, oh Ross,  
+ Ross Robinson. 
```

Comment:
Modificare in:" And she said to me in this voice, she said:"oh Ross,  
 Ross Ross" "